src/Sarif/IResultLogWriter.cs

```diff
@@ -41,15 +41,15 @@ public interface IResultLogWriter
         void WriteLogicalLocations(IList<LogicalLocation> logicalLocations);
 
         /// <summary>
-        /// Write information about rules to the log. This information may appear
-        /// after the results, as the relevant set of rules might not be known until
-        /// all results have been generated. A Sarif file may also contain only rules
-        /// metadata.
+        /// Write information about the tool to the log. This information may appear
+        /// after the results, as some tool data, such as the relevant set of rule
+        /// observed during analysis, might not be known until all results have been
+        /// generated. A Sarif file may also contain only message metadata.
         /// </summary>
-        /// <param name="rules">
-        /// An array whose elements comprise rules relevant to the analysis.
+        /// <param name="tool">
+        /// The tool object relevant to the current run.
```

Comment:
>relevant to the current run [](start = 28, length = 27)

I'd only use "X relevant to Y" when I can't find a more specific way to express the relationship between X and Y. How about "The tool that was executed in the current run".